### PR TITLE
qemu.tests.cfg: Update boot_device config for seabios

### PR DESCRIPTION
**Pull request by spcui**
ID: 1223292

### CODE REVIEW

File: qemu/tests/cfg/seabios.cfg [hunk 1]
```diff
@@ -11,6 +11,8 @@
     boot_menu_hint = "Press .*F12 for boot menu"
     # Specify the boot device name which you want to test here.
     boot_device = "iPXE"
+    Host_RHEL.6:
```
Comment (CongSmile) [inline qemu/tests/cfg/seabios.cfg]: Hi @spcui 

How about RHEL.5 host, I think we need confirm it.

Thanks.

Comment (spcui) [inline qemu/tests/cfg/seabios.cfg]: Hi @CongSmile , for RHEL.5 host, using gPXE as default value.

**Comment by CongSmile** [comment]
RHEL.5 host is disabled for this case.

Then Acked-by: Cong Li <coli@redhat.com>

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by XueqiangWei** [comment]
LGTM,
Acked by Xueqiang Wei <xuwei@redhat.com>

**Comment by xutian** [comment]
thanks @CongSmile  @suqinhuang @XueqiangWei  and @spcui. Merged

**Pull request merged**